cert/cert.go

```diff
@@ -292,3 +292,18 @@ func ParseCertAndKey(certPEM, keyPEM string) (*x509.Certificate, *rsa.PrivateKey
 	}
 	return cert, key, nil
 }
+
+// Generate public key from keyPEM string
```

Comment:
the expected form for go doc is:
// ComputePublicKey generates public key from keyPEM string